fetch_data/utils/clean_string.py

```diff
@@ -0,0 +1,14 @@
+"""Clean String"""
+
+import re
+
+
+def clean_string(input_str: str) -> str:
+    """Clean String"""
```

Comment:
```suggestion
   """
   Cleans the input string by normalizing whitespace and adjusting capitalization.
   - Collapses multiple spaces and replaces non-breaking spaces with single spaces.
   - Trims leading and trailing spaces.
   - Converts the string to uppercase if its length is less than 2.
   - Ensures the first character is capitalized while keeping the rest unchanged.
   """```

suggestion of a more descriptive comment